test/acceptance/features/steps/openshift.py

```diff
@@ -193,3 +248,44 @@ def get_resource_info_by_jq(self, resource_type, name, namespace, jq_expression,
                     time.sleep(5)
         exit_code | should.be_equal_to(0).desc(f'Exit code should be 0:\n OUTPUT:\n{output}')
         return output.rstrip("\n")
+
+    def create_image_stream(self, name, registry_namespace):
+        image_stream = self.image_stream_template.format(name=name, namespace=registry_namespace)
+        return self.oc_apply(image_stream)
+
+    def get_docker_image_repository(self, name, namespace):
+        (output, exit_code) = self.cmd.run(f'oc get is {name} -n {namespace} -o jsonpath="{{.status.dockerImageRepository}}"')
+        exit_code | should.be_equal_to(0)
```

Comment:
add `.desc(output)` so that we can see what happened.

Reply:
sure, Thanks

Reply:
`exit_code | should.be_equal_to(0).desc(f"cmd-{cmd} result for getting docket image repository is {output} with exit code-{exit_code}")`
Let me know if this is fine